package.json

```diff
@@ -30,18 +30,18 @@
     "@graphql-typed-document-node/core": "^3.1.0",
     "@types/aws4": "^1.5.1",
     "@types/jest": "^26.0.15",
-    "@types/node": "^14.14.10",
+    "@types/node": "^16.18.96",
     "@typescript-eslint/eslint-plugin": "^4.9.0",
     "@typescript-eslint/parser": "^4.9.0",
     "eslint": "^7.14.0",
     "eslint-config-airbnb-typescript": "^12.0.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-prettier": "^3.1.4",
-    "jest": "^26.6.3",
+    "jest": "^29.7.0",
```

Comment:
updating Jest was necessary to support `AbortController` in tests. At least v27 is required, I updated all the way.